### ISSUE TITLE
Repo Readme.md links to dead pages for documentation links

### ISSUE THREAD

**Issue by hux** [AppScale/appscale#3226]
Was hoping to try this out! 
Try App-Scale, Fast-Start, and all the other ones I try - hit 404's on appscale.com/get-started/
Is there another place for the documentation?

**Comment by cyberquarks** [comment]
@hux Yes you can access the documentation through https://web.archive.org/web/20180918063722/https://www.appscale.com/try-appscale

Internet Archive. 

Then just grab an HTTP page downloaded to scrape the whole website for local copy. @obino might be able to point you to a repo of the docs, IDK if that exists. 

and ps. it's best to run in docker
